### PR TITLE
ci(pr/title): PR title support new check list

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
The title of the PR should conform to mainstream community norms.
At present, there is no restriction on commit, and the corresponding rules will be added after discussion.
Header contains three fields: type (required), scope (optional), subject (required)
The constraints of these three parts in the current CI rules are as follows:
- type: must be the specified type
- scope: currently unrestricted
- subject: capitalize the first letter of the sentence

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. PR title support new check list.
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: .github/workflows/pr_title.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      # Please look up the latest version from
+      # https://github.com/amannn/action-semantic-pull-request/releases
+      - uses: amannn/action-semantic-pull-request@v3.4.6
```
Comment (ShiChangkuo) [inline .github/workflows/pr_title.yml]: the latest version is v5 already

File: .github/workflows/pr_title.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            ci
+            chore
+            refactor
+            test
+            style
+          # Configure that a scope must always be provided.
+          requireScope: false
+          # Configure additional validation for the subject based on a regex.
+          # This example ensures the subject starts with an uppercase character.
+          subjectPattern: ^[A-Z].+$
```
Comment (ShiChangkuo) [inline .github/workflows/pr_title.yml]: maybe starting with an uppercase or lowercase character is both acceptable.

File: .github/workflows/pr_title.yml (hunk 1 above)
Comment (Lance52259) [reply .github/workflows/pr_title.yml]: done. using ver.5.0.2 instead.

File: .github/workflows/pr_title.yml (hunk 2 above)
Comment (Lance52259) [reply .github/workflows/pr_title.yml]: done. the lowercase letter has been supported.

**Pull request merged**